packages/calcite-components/src/components/color-picker/color-picker.scss

```diff
@@ -3,6 +3,18 @@
  *
  * These properties can be overridden using the component's tag as selector.
  *
+ * @prop --calcite-color-picker-button-background-color-active: defines the background color of the button when in a active state in the component.
```

Comment:
These are for "add / remove saved" yeah? Maybe a descriptor in the name as it isn't immediately clear what these are used for and that functionality can be disabled via prop.